etstool.py

```diff
@@ -61,10 +61,8 @@
 Changing This File
 ------------------
 
-To change the packages installed during a test run, change the dependencies
-variable below.  To install a package from github, or one which is not yet
-available via EDM, add it to the `ci-src-requirements.txt` file (these will be
-installed by `pip`).
+To change the packages installed during a test run, change the
+``common_dependencies`` variable below.
```

Comment:
Drive-by fix: the old comment referred to a "dependencies" variable, but the code has since been updated and there isn't a "dependencies" variable any more.